### PR TITLE
Project details initial version + Outline button fix

### PR DESCRIPTION
**Pull request by mateodaza**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -29,29 +28,8 @@ export default function RootLayout({
...
-							height={109}
-							alt='arc-2'
-						/>
-						<Image
-							className='absolute top-96 right-0 -z-10'
-							src='/images/arcs/3.svg'
-							width={116}
-							height={369}
-							alt='arc-3'
-						/>
```
Comment (MohammadPCh) [inline src/app/layout.tsx:52]: please keep them in the layout, because we have them on all views.

File: src/components/Button/OutlineButton.tsx [hunk 2]
```diff
@@ -53,10 +53,10 @@ export const OutlineButton: FC<OutlineButtonProps> = ({
 		>
 			<div
 				id='shadow'
-				className='absolute w-full h-full  bg-black z-0  bottom-0 animate-move-bounce-leave transform group-hover/OutlineButton:animate-move-bounce-enter'
+				className='absolute w-full h-full bg-black z-0 bottom-0 animate-move-bounce-leave transform group-hover/OutlineButton:animate-move-bounce-enter'
 			/>
```
Comment (MohammadPCh) [inline src/components/Button/OutlineButton.tsx:57]: It was based on the animation that Mosaeedi provided. I suggest focusing on functionality and leaving the fancy stuff for the next phase. 😄

File: src/features/home/queries.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    totalFlags
+    attestedOrganisations {
+      vouch
+      organisation {
+        name
+      }
+    }
+  }
+}
+`;
```
Comment (MohammadPCh) [inline src/features/home/queries.ts:20]: We don't need this anymore

File: tailwind.config.ts [hunk 4]
```diff
@@ -29,50 +29,36 @@ const config: Config = {
...
 					'100%': {
-						backgroundColor: 'var(--bounce-bg-end, #FFFFFF)',
-						border: '1px solid var(--bounce-border-end, #000000)',
-						color: 'var(--bounce-color-end, #3742FA)',
+						backgroundColor: 'var(--bounce-bg-end)',
+						borderColor: 'var(--bounce-border-end)',
+						color: 'white', // Ensure it stays white
 					},
 				},
 			},
```
Comment (MohammadPCh) [inline tailwind.config.ts:64]: same as the outline button please 😄

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
@mateodaza Thanks for this great PR. I put my suggestions.

**Comment by MohammadPCh** [comment]
@mateodaza, please remove the `pnpm-lock.yam` since we use yarn for this project.

File: src/app/layout.tsx (hunk 1 above)
Comment (mateodaza) [reply src/app/layout.tsx:52]: Project details view don't have them

image

**Review by MohammadPCh — APPROVED** [review]
Thanks @mateodaza

**Pull request merged**